code/modules/clothing/head/helmet.dm

```diff
@@ -372,11 +372,11 @@
 	strip_delay = 80
 
 /obj/item/clothing/head/helmet/elder_atmosian
-	name = "Elder Atmosian Helmet"
+	name = "/improper Elder Atmosian Helmet"
 	desc = "A superb helmet made with the toughest and rarest materials available to man."
-	icon_state = "knight_greyscale"
-	inhand_icon_state = "knight_greyscale"
-	armor = list(MELEE = 15, BULLET = 10, LASER = 30, ENERGY = 30, BOMB = 10, BIO = 10, RAD = 20, FIRE = 65, ACID = 40, WOUND = 15)
+	icon_state = "h2helmet"
+	inhand_icon_state = "h2helmet"
+	armor = list(MELEE = 50, BULLET = 30, LASER = 40, ENERGY = 40, BOMB = 20, BIO = 10, RAD = 40, FIRE = 100, ACID = 40, WOUND = 30)
```

Comment:
No, the armour gets its values from the material. Don't buff it like this.

Reply:
To explain this more accurately, materials apply a multiplier to the armor values. If you want it to use specifically these values, remove the MATERIAL_AFFECT_STATISTICS flag from the material_flags.